dev/standalone-community/galaxy_ng.env

```diff
@@ -14,6 +14,7 @@ PULP_ANSIBLE_CONTENT_HOSTNAME=http://localhost:24816/api/v3/artifacts/collection
 PULP_TOKEN_AUTH_DISABLED=true
 PULP_CONTENT_ORIGIN="http://localhost:24816"
 
+PULP_GALAXY_DISPLAY_REPOSITORIES=false
```

Comment:
Do we need this added both as a setting and as a feature flag?

```bash
export PULP_GALAXY_FEATURE_FLAGS__display_repositories=false
```
would work in the same way

Reply:
then the hook function would not be needed